### ISSUE TITLE
The problem of imbalanced output values at different times

### ISSUE THREAD

**Issue by ShupengGao** [lululxvi/deepxde#1326]
Dear Lu，
There is a system of PDE about time and space, which contains 10 equations.
QUESTION:
The order of magnitude of each output's true value is 10 ^ (-), and there are significant differences at different times.
I tried using net.apply_output_transform(output_transform).

def output_transform(x,y):
    return torch.cat(
        [
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,0:1] * 1.2), 
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,1:2] * 10 ** -4 * 5) ,
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,2:3] * 10 ** -1),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,3:4] * 10 ** -4 * 5),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,4:5] * 10 ** -5),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,5:6] * 10 ** -5 * 5),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,6:7] * 10 ** -9 * 8),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,7:8] * 10 ** -8 * 5),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,8:9] * 16 + 3),
            abs(x[:,0:1] * (x[:,0:1] - 505) * x[:,1:2] * (x[:,1:2] - 505) / (505 ** 4) * x[:,2:3] * y[:,9:10] * 0.5 + 0.3),
        ],
        dim=1,
    )

However,this scaling is specific to a certain moment(t=0.5), and the order of magnitude at other moments may change.All values will decrease, even by 10 ^ 2 times(t=0.1) and all values will increase, even by 10 ^ 2 times(t=1).
Therefore, I want to balance the range of 10 output values at all times. What should I do?

**Comment by lululxvi** [comment]
1. Use `loss_weights` https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#deepxde.model.Model.compile
2. You can update `loss_weights` every certain iterations.

**Comment by ShupengGao** [comment]
> 1. Use `loss_weights` https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#deepxde.model.Model.compile
> 2. You can update `loss_weights` every certain iterations.

Dear Lu,
Thanks for your answer.

I may need to describe my problem in more detail:
(1)Is the network unfriendly to output values to 10^(-),such as 10^(-9) and Unbalanced output?
(2)Assuming the output is 3*10^(-6),then the calculation result of loss function is about 10^(-12),if loss function is MSE.
This will result in incorrect final results
(3)For different times t, the magnitude of the output value varies. How should I use loss_ weights.

**Comment by lululxvi** [comment]
> (1)Is the network unfriendly to output values to 10^(-),such as 10^(-9) and Unbalanced output? 

Yes.

> (2)Assuming the output is 3*10^(-6),then the calculation result of loss function is about 10^(-12),if loss function is MSE. This will result in incorrect final results

You are right.

> (3)For different times t, the magnitude of the output value varies. How should I use loss_ weights.

`loss_ weights` is used to weight different loss terms.
